src/SFA.DAS.Reservations.Api.AcceptanceTests/CreateReservationSteps.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+using SFA.DAS.Common.Domain.Types;
+using SFA.DAS.Reservations.Api.Controllers;
+using SFA.DAS.Reservations.Data;
+using SFA.DAS.Reservations.Domain.Entities;
+using TechTalk.SpecFlow;
+using Course = SFA.DAS.Reservations.Domain.Entities.Course;
+using Reservation = SFA.DAS.Reservations.Api.Models.Reservation;
+
+namespace SFA.DAS.Reservations.Api.AcceptanceTests
+{
+    [Binding]
+    public class CreateReservationSteps
+    {
+        private readonly TestData _testData;
+        private readonly IServiceProvider _services;
+
+        public CreateReservationSteps(TestData testData, TestServiceProvider serviceProvider)
+        {
+            _testData = testData;
+            _services = serviceProvider;
+        }
+
+        [Given(@"I have a non levy account")]
+        public void GivenIHaveANonLevyAccount()
+        {
+            _testData.IsLevyAccount = false;
+        }
+
+        [Given(@"I have a levy account")]
+        public void GivenIHaveALevyAccount()
+        {
+            _testData.IsLevyAccount = true;
+        }
+
+        [Given(@"a course name (.*) has been added to the course list")]
+        public void GivenACourseNameHasBeenAddedToTheCourseList(string courseName)
```

Comment:
This seems to be doing a bit more than whats suggested - I think that all of this given should exist outside of the scenario.